tests/protos/test_fully_connected.py

```diff
@@ -22,19 +23,30 @@ def fully_connecteds(
     st.SearchStrategy[Tuple[fully_connected_pb2.FullyConnected, Dict]],
 ]:
     """Returns a SearchStrategy for a FC layer plus maybe the kwargs."""
-    kwargs = {}
+    eps = 1e-8
 
+    kwargs = {}
+    to_ignore = []
```

Comment:
Can `kwargs["dropout"]` be set to `None` rather than being added to `to_ignore` in the invalid case (i.e. remove `to_ignore` entirely)? `None` is the default value of the field when not set

Reply:
Yeah good point. done